run.sh

```diff
@@ -27,7 +33,16 @@ psql "$DATABASE_URL" -f generated/update-spark-db.sql | tee generated/dbupdate.l
 echo "** Updating client-allocator mappings **"
 node scripts/update-allocator-clients.js | tee generated/allocator-update.log
 
-echo "** FINISHED INGESTION OF f05 DEALS**"
+MESSAGE=$(
+echo "**FINISHED INGESTION OF f05 DEALS**"
 grep "^DELETE" < generated/dbupdate.log | awk '{s+=$2} END {print "Deleted: " s}'
 grep "^INSERT" < generated/dbupdate.log | awk '{s+=$3} END {print "Added: " s}'
 tail -1 generated/allocator-update.log
+)
+
+if [ -n "$SLACK_WEBHOOK_URL" ]; then
+  echo "** Sending message to slack **"
+  curl -X POST -H 'Content-type: application/json' --data "{\"text\":\"$MESSAGE\"}" $SLACK_WEBHOOK_URL
+else 
+  echo $MESSAGE
+fi
```

Comment:
Let's always print the summary to stdout, it will make it easier for us to find this information when inspecting logs in papertrail.

```suggestion
echo $MESSAGE

if [ -n "$SLACK_WEBHOOK_URL" ]; then
  echo "** Sending message to slack **"
  curl -X POST -H 'Content-type: application/json' --data "{\"text\":\"$MESSAGE\"}" $SLACK_WEBHOOK_URL
fi
```